.github/workflows/ci_tests.yml

```
@@ -43,7 +43,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.7", "3.8", "3.9"]
+        python-version: ["3.7", "3.8", "3.10"]
```

```suggestion
        python-version: ["3.7", "3.8", "3.9", "3.10"]
```
Why remove testing Python 3.9??